src/translators/image_translator.py

```diff
@@ -16,9 +17,13 @@
 from azure.core.credentials import AzureKeyCredential
 from src.config.base_config import Config
 from src.translators.text_translator import TextTranslator
+from src.utils.file_utils import generate_translated_filename
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
```

Comment:
I'll adjust the logging configuration by removing `logger.setLevel(logging.DEBUG)` to allow users to set their own logging preferences.